index.html

```diff
@@ -61,6 +61,282 @@ <h1>i'm Adan <br>
             </ul>
 
         </section>
+
+        <section class="works">
+            <div class="grid1">
+                <img src="./images/Snapshoot Portfolio.png" alt="snapshot">
+
+            </div>
+            <div class="details">
+                <h1>Tonic</h1>
+
+                <ul class="specific">
+                    <li class="images">CANOPY</li>
+                    <li id="fade">Back End Dev</li>
+                    <li id="fade">2015</li>
+                </ul>
+
+                <p class="brief">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+
+                <div class="languages">
+                    <ul class="html-css">
+                        <li>
+                            <a href=""><img src="./images/html.png" class="img2" alt=""></a>
+                        </li>
+                        <li>
+                            <a href="#index.html"><img src="./images/Tag.png" class="img2" alt=""></a>
+
+                        </li>
+                        <li>
+                            <a href="#index.html"><img src="./images/Tag (1).png" class="img2" alt=""></a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="see-projects">
+                    <button class="see">See Projects</button>
+                </div>
+
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly make sure all the **"See Projects"** buttons and the **"Get My Resume"** follow the button styling: Active, Hover, and Pressed (see design) as in the image below. You can check all details from the Figma design.
![Screenshot from 2022-09-24 16-10-54](https://user-images.githubusercontent.com/83042742/192102718-6b1ec211-c25f-4367-a41d-e9335f7cabff.png)